turtle_tf2_py/turtle_tf2_py/turtle_tf2_listener.py

```diff
@@ -43,6 +43,7 @@ def __init__(self):
             self.get_logger().info('service not available, waiting again...')
 
         # Initialize request with turtle name and coordinates
+        # Note that x, y and theta are defined as floats in turtlesim/srv/Spawn
         request = Spawn.Request()
         request.name = 'turtle2'
         request.x = float(4)
```

Comment:
You can do this: 
```python
request.x = float(4)
# or
request.x = 4.0
```

Reply:
Do you mean that I should replace typecasting with adding a floating point on these 3 values?

Reply:
not necessarily, I was just adding a comment, it more a Python related thing, maybe it does not make sense to do nothing